.github/scripts/dapr_bot.js

```diff
@@ -55,11 +87,52 @@ module.exports = async ({ github, context }) => {
             await cmdOkToPerf(github, issue, isFromPulls);
             break;
         default:
-            console.log(`[main] command ${command} not found, exiting.`);
+            console.log(`[handleIssueCommentCreate] command ${command} not found, exiting.`);
             break;
     }
 }
 
+/**
+ * Handle issue labeled event.
+ */
+async function handleIssueLabeled({ github, context }) {
+    const payload = context.payload;
+    const label = payload.label.name;
+    const issueNumber = payload.issue.number;
+
+    // This should not run in forks.
+    if (context.repo.owner !== "dapr") {
```

Comment:
What if a fork removes this line, would they be able to run this task?

Reply:
They don't have access to the bot credentials.. we do not have credential sharing enabled. So no, they can't really run this.